lightwood/encoder/audio/mfcc.py

```diff
@@ -9,20 +9,40 @@
 
 class MFCCEncoder(BaseEncoder):
     """
-    Audio encoder. Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing. The features are a 2D array, flattened into a 1D one.
+    Audio encoder.
+
+    Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing.
```

Comment:
might be worth linking a paper or even wiki if it's a standard enough technique. I am not familiar with audio processing approaches.